src/js/components/Home/Countries.js

```diff
@@ -2,22 +2,44 @@ import React, {Component} from "react";
 import Map from "./Map";
 import './MapStyles.sass'
 import ReactFlagsSelect from 'react-flags-select';
+import Geocode from "react-geocode";
 
+// function onSelectFlag(countryCode){             //TODO: Function to be implemented
+//     console.log(countryCode)
+// }
+// set Google Maps Geocoding API for purposes of quota management. Its optional but recommended.
+Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
 
 class Mapbuttons extends Component {
     constructor() {
```

Comment:
Your constructor does not make use of props as its parameter and the same the same applies for super()
If you need to access this.props in your constructor, you'll need props and knowing that your component is not stateless, there are better chances that you will need one